xformers/ops/swiglu.py

```diff
@@ -0,0 +1,127 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All rights reserved.
+#
+# This source code is licensed under the BSD license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import Optional
+
+import torch
+import torch.nn.functional as F
+from torch import nn
+
+
+class _SwiGLUModule(nn.Module):
```

Comment:
If we want to expose this to the user through other ops, we can make this public as well, and expose an `op` argument.
```suggestion
class _SwiGLU(nn.Module):
```